js_modules/dagster-ui/packages/ui-core/src/ui/Filters/__tests__/useTimeRangeFilter.test.tsx

```diff
@@ -206,7 +206,7 @@ describe('ActiveFilterState', () => {
       />,
     );
 
-    expect(getByText(/Timestamp is after/)).toBeInTheDocument();
+    expect(getByText(/Created after/)).toBeInTheDocument();
```

Comment:
I'm a little wary of this wording because it may not be the case that every instance of this filter is referring to something being created a specific time

Reply:
I think it would be better if the filter itself made the language here customizable and we only changed in this instance for now.

Reply:
Ok I added a new prop to control this so you can choose what the term is to match the `name` of the filter